crates/holochain_p2p/src/spawn/actor.rs

```diff
@@ -55,8 +63,9 @@ impl WrapEvtSender {
         peer_data: Vec<AgentInfoSigned>,
     ) -> impl Future<Output = HolochainP2pResult<Vec<AgentInfoPut>>> + 'static + Send {
         timing_trace!(
+            false,
             { self.0.put_agent_info_signed(dna_hash, peer_data) },
-            "(hp2p:handle) put_agent_info_signed",
+            a = "recv_put_agent_info_signed",
```

Comment:
Why `a =`?

Reply:
Ahh okay, that makes sense now! I wonder if it's worth adding some docs somewhere about enabling this tracing and also mention what the letters refer to? It's useful for people reading the logs as well as for me if I'm adding something new and trying to remember where I asked the question already :)